editor/package_dialog.py

```
@@ -0,0 +1,133 @@
+
+class PackageMetadataDialog(QDialog):
+    """
+    Collects package metadata from the author before export.
+    Validates required fields and provides banner image selection.
+    """
+    
+    def __init__(self, current_map_path: str, parent=None):
+        super().__init__(parent)
+        self.setWindowTitle("Export Game Package")
+        self.setMinimumWidth(500)
+        self._banner_path: Optional[str] = None
```

Ruff (0.15.12)

[error] Undefined name `Optional`

(F821)